sample-xml/domain/src/main/java/co/nimblehq/sample/xml/domain/usecase/IsFirstTimeLaunchPreferencesUseCase.kt

```diff
@@ -0,0 +1,14 @@
+package co.nimblehq.sample.xml.domain.usecase
+
+import co.nimblehq.sample.xml.domain.repository.AppPreferencesRepository
+import kotlinx.coroutines.flow.Flow
+import javax.inject.Inject
+
+class IsFirstTimeLaunchPreferencesUseCase @Inject constructor(
```

Comment:
Could you please add tests for these 2 use-cases? Basically copy them from compose-templates.

Reply:
@longnd  Resolved in 08943cfc